docker-angor-api/docker-compose.yml

```diff
@@ -53,16 +67,23 @@ services:
       DATABASE_USERNAME: "mempool"
       DATABASE_PASSWORD: "mempool"
       STATISTICS_ENABLED: "true"
-    user: "1000:1000"
+      VIRTUAL_HOST: indexer.thedude.pro
```

Comment:
The only comment I have is that we should use the `indexer.angor.io` domain on the main repo

Reply:
```suggestion
      VIRTUAL_HOST: indexer.angor.io
```